api/desecapi/serializers.py

```diff
@@ -16,6 +18,10 @@
 from desecapi import models
 
 
+SECRET_KEY_ENCRYPT = urlsafe_b64encode(sha256(('encrypt-' + settings.SECRET_KEY).encode()).digest())  # DO NOT MODIFY
```

Comment:
duplicate code

Reply:
use serious kdf, https://csrc.nist.gov/publications/detail/sp/800-108/final

Reply:
I read SP 800-108 and then worked out two ways to do this, see https://github.com/desec-io/desec-stack/pull/263/commits/6bd669eb48a0bc7ebd07fcebb2da665f523a6678.

As the next step, I'd like to discuss them with you in person and point out some questions I have.